src/components/name-input.js

```diff
@@ -10,18 +10,30 @@ import {
 } from '../styles/pages/welcome';
 import ArrowIcon from '../icons/white-arrow.png';
 
-export const NameInputView = ({ nextPage }) => (
-  <NameInputContainer>
-    <TextInputWrapper>
-      <NameInputText placeholder="What&apos;s your name?" />
-    </TextInputWrapper>
-    <NextButton onPress={nextPage}>
-      <ButtonText>Next</ButtonText>
-      <WhiteArrowIcon source={ArrowIcon} />
-    </NextButton>
-  </NameInputContainer>
-);
+export default class NameInputView extends React.Component {
```

Comment:
Prefer a refactor commit that converts a stateless component function into a component class and then make your changes after (add constructor, state, and new function).

Reply:
done @cbarber

Reply:
Almost. You should keep the original export structure. Looks like the import is using the named export and not the default.

Feature commit looks great now